src/sentry-algolia-index-settings.ts

```diff
@@ -14,22 +14,24 @@ export const settings: Settings = {
   attributesToHighlight: ['section'],
   attributesForFaceting: [
     'filterOnly(pathSegments)',
-    'filterOnly(platforms)',
+    'filterOnly(sdk)',
+    'filterOnly(framework)',
     'filterOnly(legacy)',
   ],
   distinct: 1,
   attributeForDistinct: 'section',
-  attributesToRetrieve: [
-    'text',
-    'title',
-    'section',
-    'url',
-    'anchor',
-    'context',
-  ],
+  attributesToRetrieve: ['*'],
   searchableAttributes: ['unordered(section)', 'unordered(keywords)', 'text'],
-  customRanking: ['asc(sectionRank)', 'desc(position)'],
```

Comment:
I suspect if we don't update the other sites that use sentry-global-search, like the blog & help they'll keep adding back these incorrect `customRanking` attributes to Algolia every time they publish to master...